libs/external-router/routing/src/matcher/daff-type-matcher.ts

```diff
@@ -0,0 +1,6 @@
+import {
+  UrlMatcher,
+  UrlSegment,
+} from '@angular/router';
+
+export const daffExternalRouterTypeMatcher: UrlMatcher = (url: UrlSegment[]) => ({ consumed: url });
```

Comment:
Document this type. In particular, address questions such as "Why does this exist?", "As an app dev, when and why would I need to use this?"

Reply:
I deleted this. It was mistakenly added.